src/spotted/data/user.py

```diff
@@ -95,6 +106,40 @@ def sban(self) -> bool:
             return True
         return False
 
+    def mute(self, bot: Bot, days: int = 0):
+        """_summary_
+
+        Args:
+            bot (Bot): the telegram bot
+            days (int, optional): The number of days the user should be muted for.
+                                    Defaults to 0.
```

Comment:
Default docstring and no need to specify the types of the parameters. Sphinx is able to infer them automatically